### PR TITLE
Add a link to the event signup form

### PR DESCRIPTION
**Pull request by johannhof**
This adds a link to a Google Form that posts emails to the community-berlin-peers list. We prepared that form and the mailing list at the Open Mozilla Night tonight. It's currently a bit simple, I'll work on improving the automated email messages later. It should be enough to get us rolling.

Can you all take a quick look and sign this off, please? Feel free to nitpick on the wording in this PR.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,6 +17,9 @@ <h2 class="h2 left-align white">Mozilla Berlin</h2>
         <h2 class="center">Berlin <span class="mozilla--title">moz://a</span> Community Space</h2>
         <p class="center">GSG-Hof Schlesische Straße Gebäude 3, 4th floor<br> Schlesische Straße 27<br> 10997 Berlin</p>
       </div>
+      <h2 class="mozilla--title">Events</h2>
+      <p>We regularly host tech meetups, learning groups and other web-related gatherings, free of charge. Our space is sponsored and maintained by Mozilla, but we operate based on a trust model that enables volunteers to indepently host their own events.</p>
+      <p><strong>Do you want to run an event in the Mozilla Community Space? Please get in touch <a href="https://goo.gl/forms/4fZZzBB9QnCUemyd2">here</a>!</strong></p>
```
Comment (caugner) [inline index.html:22]: *Would you like* to run...? (or) *Interested in running*...?

File: index.html [hunk 2]
```diff
@@ -17,6 +17,9 @@ <h2 class="h2 left-align white">Mozilla Berlin</h2>
         <h2 class="center">Berlin <span class="mozilla--title">moz://a</span> Community Space</h2>
         <p class="center">GSG-Hof Schlesische Straße Gebäude 3, 4th floor<br> Schlesische Straße 27<br> 10997 Berlin</p>
       </div>
+      <h2 class="mozilla--title">Events</h2>
```
Comment (caugner) [inline index.html:20]: Double dash in h2 class intentionally?

Comment (caugner) [reply index.html:20]: Ah, consistent with others.

**Review by caugner — COMMENTED** [review]
Thanks, looking very good, I'm just not a big fan of "wanting".

File: index.html (hunk 1 above)
Comment (fiji-flo) [reply index.html:22]: I like this very straightforward approach. I would approve this. We can change it at any time.

**Review by fiji-flo — APPROVED** [review]
thanks for updating. 👍

**Pull request merged**